File: frontend/src/app/dashboard/model/dto/chart-creation-options.dto.ts
export interface ChartCreationOptionsDto {
  chartCreationOptionsId?: number;
  title: string;
  chartType: ChartInformationTypeEnum;
  selectedTeamIds?: number[];
}

export enum ChartInformationTypeEnum {
  LINE_PROGRESS,
  PIE_TOPICDRAFTOVERVIEW,
}

export const ChartTypeEnumDropDownRecord: Record<ChartInformationTypeEnum, string> = {
  [ChartInformationTypeEnum.LINE_PROGRESS]: 'edit-dashboard.line-progress',
  [ChartInformationTypeEnum.PIE_TOPICDRAFTOVERVIEW]: 'edit-dashboard.pie-topicdraft',
};

File: backend/burning-okr/burning-okr-model/src/main/java/org/burningokr/model/dashboard/creation/ChartCreationOptionsDto.java
package org.burningokr.model.dashboard.creation;

import lombok.Data;
import org.burningokr.model.dashboard.ChartInformationTypeEnum;

import java.util.Collection;

@Data
public class ChartCreationOptionsDto {
  private Long id;
  private String title;
  private ChartInformationTypeEnum chartType;
  private Collection<Long> selectedTeamIds;
}
